resources/opensearchservice-packages.go

```diff
@@ -0,0 +1,53 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/opensearchservice"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type OSPackage struct {
+	svc       *opensearchservice.OpenSearchService
+	packageID *string
+}
+
+func init() {
+	register("OSPackage", ListOSPackages)
+}
+
+func ListOSPackages(sess *session.Session) ([]Resource, error) {
+	svc := opensearchservice.New(sess)
+
+	listResp, err := svc.DescribePackages(&opensearchservice.DescribePackagesInput{})
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+
+	for _, pkg := range listResp.PackageDetailsList {
+		resources = append(resources, &OSPackage{
+			svc:       svc,
+			packageID: pkg.PackageID,
+		})
+	}
+
+	return resources, nil
+}
+
+func (o *OSPackage) Remove() error {
+	_, err := o.svc.DeletePackage(&opensearchservice.DeletePackageInput{
+		PackageID: o.packageID,
+	})
+
+	return err
+}
+
+func (o *OSPackage) Properties() types.Properties {
+	properties := types.NewProperties()
+	return properties
+}
```

Comment:
We should either include additional properties here or remove the function entirely to prevent printing out an empty properties array. It appears [PackageDetails](https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.44.300/service/opensearchservice#PackageDetails) gives us some additional metadata about the package. It would be nice to include at least `PackageName`, `CreatedAt` for more informative output. If it helps, I had to [do the same in my most recent Polly Lexicon PR](https://github.com/oreillymedia/aws-nuke/pull/5/commits/da4aff3b636c85decb1e00319a92d4ba3346ed94).

Reply:
Done!